docs/7___appendix.adoc

```diff
@@ -283,11 +283,25 @@ The FMI 3.0 standard text is based on FMI 2.0 and FMI 1.0 and we wish to acknowl
 
 Special thanks to
 
-- Andreas Junghanns (Synopsys) and Torsten Sommer (Dassault Syst&#232;mes) for the initial conversion of the FMI 2.0 Word document to AsciiDoc
-- Torsten Sommer (Dassault Syst&#232;mes) for the creation of the reference FMUs and the github CI tool chain
-- Cl&#225;udio Gomes (Aarhus University) for the creation of the tooling to create the graphical representation of the XML schemas
-- Andreas Junghanns (Synopsys) and Torsten Blochwitz (ESI Group) for restructuring and re-writing the whole FMI standard text
-- all companies and organizations that participated in the FMI 3.0 Plugfests: Aarhus University, Altair, Augsburg University, AVL, Dassault Syst&#232;mes, dSPACE, ESI Group, ETAS, Julia Computing, Maplesoft, OSMC, PMSF IT Consulting, Synopsys, TLK
+* Andreas Junghanns (Synopsys) and Torsten Sommer (Dassault Syst&#232;mes) for the initial conversion of the FMI 2.0 Word document to AsciiDoc
+* Torsten Sommer (Dassault Syst&#232;mes) for the creation of the reference FMUs and the github CI tool chain
+* Cl&#225;udio Gomes (Aarhus University) for the creation of the tooling to create the graphical representation of the XML schemas
+* Andreas Junghanns (Synopsys) and Torsten Blochwitz (ESI Group) for restructuring and re-writing the whole FMI standard text
+* all companies and organizations that participated in the FMI 3.0 Plugfests:
+** Aarhus University with VDMCheck
+** Altair with Activate
+** Augsburg University with FMI.jl
+** AVL List GmbH with ModelConnect
+** Dassault Syst&#232;mes with Dymola, FMPy, FMI-Kit
+** dSPACE GmbH with TargetLink, SystemDesk, Veos
```

Comment:
dSPACE GmbH with TargetLink, SystemDesk, ConfigurationDesk and VEOS

Reply:
Done.